layers/timeline_semaphore.c

```diff
@@ -543,10 +543,12 @@ static VkResult device_new(VkDevice _device,
     if (pthread_condattr_init(&condattr) != 0)
         goto err;
 
+#if !defined(__APPLE__)
     if (pthread_condattr_setclock(&condattr, CLOCK_MONOTONIC) != 0) {
```

Comment:
Do you not need this at all on MacOS?

Reply:
That function doesn't exist, so I'm assuming this is because there is only one clock domain there.